### PR TITLE
Cpt wp global styles fixes

### PR DESCRIPTION
**Pull request by kebbet**
Trac ticket: https://core.trac.wordpress.org/ticket/54336

Adresses https://core.trac.wordpress.org/ticket/54336#comment:13 and https://core.trac.wordpress.org/ticket/54336#comment:14

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by oandregal — APPROVED** [review]
Changes make sense to me. Thanks for the follow up.

**Comment by oandregal** [comment]
@kebbet would you remove the changes for the calendar block? It's unrelated to global styles and it's causing some tests to fail. That may need a separate PR where we can investigate in isolation, while the global styles changes can already be merged.

**Comment by kebbet** [comment]
> @kebbet would you remove the changes for the calendar block? It's unrelated to global styles and it's causing some tests to fail. That may need a separate PR where we can investigate in isolation, while the global styles changes can already be merged.

@oandregal Reverted.

**Comment by kebbet** [comment]
And a separate PR for calendar block: https://github.com/WordPress/wordpress-develop/pull/1845

**Comment by swissspidy** [comment]
@jorgefilipecosta FYI

File: src/wp-includes/post.php [hunk 1]
```diff
@@ -449,9 +449,10 @@ function create_initial_post_types() {
 	register_post_type(
 		'wp_global_styles',
 		array(
-			'label'        => __( 'Global Styles', 'gutenberg' ),
-			'description'  => 'CPT to store user design tokens',
+			'label'        => _x( 'Global Styles', 'post type general name' ),
+			'description'  => __( 'Custom post type to include global styles in themes.' ),
```
Comment (swissspidy) [inline src/wp-includes/post.php:453]: Maybe just 

```suggestion
			'description'  => __( 'Global styles to include in themes.' ),
```

File: src/wp-includes/post.php [hunk 2]
```diff
@@ -477,28 +478,28 @@ function create_initial_post_types() {
...
 				// Some of these are a bit weird, what are they for?
-				'filter_items_list'     => __( 'Filter Navigation Menu list', 'gutenberg' ),
-				'items_list_navigation' => __( 'Navigation Menus list navigation', 'gutenberg' ),
-				'items_list'            => __( 'Navigation Menus list', 'gutenberg' ),
+				'filter_items_list'     => __( 'Filter Navigation Menu list' ),
+				'items_list_navigation' => __( 'Navigation Menus list navigation' ),
+				'items_list'            => __( 'Navigation Menus list' ),
 			),
-			'description'           => __( 'Navigation menus.', 'gutenberg' ),
+			'description'           => __( 'Custom post type for navigation menus.' ),
```
Comment (swissspidy) [inline src/wp-includes/post.php:502]: Really no reason to mention "custom post type" here.

```suggestion
			'description'           => __( 'Navigation menus.' ),
```

could even just omit it, tbh

**Review by swissspidy — CHANGES_REQUESTED** [review]

**Comment by desrosj** [comment]
I believe all of this is now merged in to WP trunk as of https://core.trac.wordpress.org/log/?revs=52106%2C52108-52110%2C52128. Please let me know if anything was left out!

Apologies again that I missed this PR initially and left out props. Thank you all for these contributions!

**Pull request closed without merging**